README.md

```diff
@@ -1 +1,124 @@
-# .github
\ No newline at end of file
+
+# Repositories: 
+- Service
+- Purpose
+- Language
+- External Integration
+- Database
+- Infrastructure
+
+
+## org: ubiquibot
+
+### 1. ubiquibot/conversation-rewards
+- Service: Conversation Reward Module
+- Purpuse: This is intended to be the proper implementation of comment incentives, based on our learnings from the first go-around.
+- Language: TypeScript
+- External Integration: Github Action, Github API
+- Database: None
+
+
+### 2. ubiquibot/permit-generation
+- Service: Permit Generation module
+- Purpuse: The purpose of this module is to isolate the bot's payment capabilities. This can be directly invoked from other plugins
+- Language: TypeScript
+- External Integration: ENS
+- Database: Supabase
+
+### 3. ubiquibot/assistive-pricing
+- Service: Assistive Pricing module
+- Purpose: **Don't know**
+- External Integration: 
+- Database: Supabase
+
+### 4. ubiquibot/comment-incentives
+
+**Deprecated in favor of @ubiquibot/conversation-rewards**
+
+- Service: Comment Incentives module
+- Purpose: Split reward to the collaborators who wrote some comments
+- Language: Typescript
+- External Integration: 
+- Database: None (Supabase, TODO: get the assignee multiplier from the database)
+
+### 5. ubiquibot/plugins-wishlist
+- Service: not a service
+- Purpose: Wishlist of the future plugins
+
+### 6. ubiquibot/configuration
+- Service: not a service
+- Purpose: Helpers to parse and merge configuration files for Ubiquibot.
+
+### 5. ubiquibot/e2e-tests
+- Service: not a service
+- Purpose: Use this for end-to-end tests.
+@pavlovcik How does it work?
+
+### 6. ubiquibot/sandbox
+- Service: Sandbox Environment
+- Purpose: This is a sandbox environment for new users to try out the bot.
+  - Try setting one Priority and one Time label to see the assistive pricing capabilities.
+  - Try the bot commands as a comment in an issue.
```

Comment:
There is a link to this one on our ubq.fi home page. I am hoping to make interacting with the bot more attractive for marketing purposes but the current implementation seems not great. I might take this down in favor of a webapp more rich experience explaining its capabilities.